core/src/time.ts

```diff
@@ -0,0 +1,39 @@
+import memoizee from 'memoizee';
```

Comment:
The naming of this module + its main export `getCurrentDate` could be clearer - without enough context it's not clear what _current_ means here